examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
> Unfortunately I can't think of how to prevent users from making mistakes
from misunderstanding these issues without otherwise blocking legitimate
use cases...

This seems correct to me - that's why I was thinking either raise a warning or have an example to illustrate what happens when you apply the SSP matrix.

Or maybe there could be a check for specific operations done on the data when `apply_proj` is called. E.g.:

```python
if raw.has_been_filtered:
    raise_warning_about_projecting_after_filtering
```